app/assets/stylesheets/primary_source_sets.css

```diff
@@ -55,6 +38,21 @@
 
 /* SourceSets index styles */
 
+.set .title-outer-container {
+  background-color: #6592A6;
+  margin-bottom: 2em;
+}
+
+.set .title-inner-container {
+  color: white;
+  padding: 1.5em;
+}
+
+.set h1 {
+  color: white;
+  margin: 0;
+}
```

Comment:
I moved this CSS so that it would be under the `/* SourceSets index styles */` header instead of the `/* SourceSet and Guide shared styles */`.